x-pack/plugins/apm/server/lib/services/get_service_transaction_groups/get_timeseries_data_for_transaction_groups.ts

```diff
@@ -76,6 +77,9 @@ export async function getTimeseriesDataForTransactionGroups({
             { term: { [SERVICE_NAME]: serviceName } },
             { term: { [TRANSACTION_TYPE]: transactionType } },
             { range: rangeFilter(start, end) },
+            ...getDocumentTypeFilterForAggregatedTransactions(
```

Comment:
When do we need `getDocumentTypeFilterForAggregatedTransactions `? Looking at the name I'd think in every request. But looking at the implementation it looks like it's only relevant in combination with `getTransactionDurationFieldForAggregatedTransactions` is that correct?

Perhaps it should be called `getTransactionDurationFilterForAggregatedTransactions`

Reply:
Btw. for consistency, wdyt about updating `getDocumentTypeFilterForAggregatedTransactions` to this:

```ts
export function getTransactionDurationFilterForAggregatedTransactions(
  searchAggregatedTransactions: boolean
) {
  return {
    exists: {
      field: searchAggregatedTransactions
        ? TRANSACTION_DURATION_HISTOGRAM
        : TRANSACTION_DURATION,
    },
  };
}
```

Reply:
Actually, why stop there:

```ts
export function getDocumentTypeFilterForAggregatedTransactions(
  searchAggregatedTransactions: boolean
) {
  return {
    exists: {
      field: getTransactionDurationFieldForAggregatedTransactions(
        searchAggregatedTransactions
      ),
    },
  };
}
```

Reply:
It would add an exists query that isn't necessary (as processor.event:transaction guarantees transaction.duration.us to exist)